packages/pds/src/api/com/atproto/server/createAccount.ts

```diff
@@ -262,12 +323,19 @@ const getDidAndPlcOp = async (
 
 // @TODO this implementation is a stub
 const assignPds = async (ctx: AppContext) => {
+  if (!ctx.cfg.service.isEntryway) return
   const pdses = await ctx.db.db.selectFrom('pds').selectAll().execute()
   const idx = randomIndexByWeight(pdses.map((pds) => pds.weight))
   if (idx === -1) return
   return pdses.at(idx)
 }
 
+const reserveSigningKey = async (host: string) => {
+  const agent = new AtpAgent({ service: getPdsEndpoint(host) })
```

Comment:
it'd be better if we only instantiate the agent once since it can be kinda expensive

but not the end of the world rn